daffodil-cli/build.sbt

```diff
@@ -71,3 +78,188 @@ rpmRelease := {
 rpmLicense := Some(licenses.value.map { case (n: String, _) => n }.mkString(" and "))
 
 rpmPrefix := Some(defaultLinuxInstallLocation.value)
+
+//
+// Windows configuration
+//
+
+//
+// Here we set the variables that are supported by the SBT Native Packager plug-in.
+// We also get fairly aggressive in editing/modifying the XML in order
+// to control and use some specific features that are supported by WiX
+// but which are not properly suported by the SBT plug-in.
+//
+
+// Force the correct installation directory name. This overwrites
+// 'daffodil-cli', which is the directory that we invoke sbt in.
+// The SBT WiX plug-in incorrectly assumes that the directory of
+// invocation is the same name as the direcotry you eventually
+// want to install into.
+name in Windows := "Daffodil" 
+
+// The Windows packager SBT plug-in maps the maintainer variable into
+// the WiX ManufacturerFullName field. This is a very strange choice.
+// It also maps it to the manufacturer name in the installation
+// directory heirarchy. Among other things, this means it needs to be
+// short so as to not blow out the path length.
```

Comment:
I think this comment may be inaccurant now? This field is no longer used in the installation directory hierarchy, so there are no really issues with length inregards to that. I beleive this field only shows up in the File > Properties > Details tab?

Reply:
Corrected.